src/game/game.html

```diff
@@ -1,53 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Battleships game</title>
-</head>
-<body>
+  </head>
+  <body>
     <main class="wrapper">
```

Comment:
It should be changed. The `wrapper` should be a `<div>` and the element with class `main` should be `<main>